pvlib/iotools/psm3.py

```diff
@@ -0,0 +1,174 @@
+
+"""
+Get PSM3 TMY
+see https://developer.nrel.gov/docs/solar/nsrdb/psm3_data_download/
+"""
+
+import io
+import requests
+import pandas as pd
+# Python-2 compatible JSONDecodeError
+try:
+    from json import JSONDecodeError
+except ImportError:
+    JSONDecodeError = ValueError
+
+URL = "http://developer.nrel.gov/api/solar/nsrdb_psm3_download.csv"
+
+# 'relative_humidity', 'total_precipitable_water' are not available
+ATTRIBUTES = [
+    'air_temperature', 'dew_point', 'dhi', 'dni', 'ghi', 'surface_albedo',
+    'surface_pressure', 'wind_direction', 'wind_speed']
+PVLIB_PYTHON = 'pvlib python'
+
+
+def get_psm3(latitude, longitude, api_key, email, names='tmy', interval=60,
+             full_name=PVLIB_PYTHON, affiliation=PVLIB_PYTHON):
```

Comment:
does sphinx render this as `full_name='pvlib python'` or `full_name=PATH_TO_SOME_OBJECT`? If the former, ok as is. If the latter, better to be repeat the string for the two default values.

Reply:
Yes,[it](https://mikofski-pvlib-python.readthedocs.io/en/patch-1/generated/pvlib.iotools.get_psm3.html) renders as "pvlib python" for both. Yoohoo! 🕺